modules/administration-guide/pages/importing-untrusted-tls-certificates.adoc

```diff
@@ -34,23 +34,24 @@ If the cluster is configured with an intermediate certificate, then the whole ch
 
 == Adding new CA certificates into {prod-short}
 
-The following procedure is applicable for already installed and running instances and for instances that are to be installed.
+Certificate files are typically stored as Base64 files, with extensions such as `.pem`, `.crt`, `.ca-bundle` and others. All Secrets that hold certificate files should use the Base64-encoded certificate rather than binary-encoded. The following procedure is applicable for already installed and running instances and for instances that are to be installed.
 
 .Prerequisites
 
 * An active `{orch-cli}` session with administrative permissions to the destination {orch-name} cluster. See {orch-cli-link}.
 * Namespace for {prod-short} exists.
+* {prod-short} already uses some reserved file names to automatically inject certificates into the ConfigMap, so you should avoid using the following reserved file names to save your certificates:
```

Comment:
```suggestion
* {prod-short} already uses some reserved file names to automatically inject certificates into the ConfigMap, so you should avoid using the following reserved file names to save your certificates:
```
```suggestion
* {prod-short} already uses some reserved file names to automatically inject certificates into the config map, so you should avoid using the following reserved file names to save your certificates:
```

Reply:
Kubernetes documentation uses "ConfigMap" spelling: https://kubernetes.io/docs/concepts/configuration/configmap/

Can I keep it?